setup.py

```diff
@@ -21,7 +21,7 @@
 from setuptools.command.build_ext import build_ext
 
 if sys.version_info[:2] < (3, 5):
-    raise Exception('This version of gensim needs 3.5 or later.')
+    raise Exception('This version of gensim needs Python 3.5 or later.')
```

Comment:
please use `setup(...., python_requires='>=3.5', ...)`, instead of raising exceptions in `setup.py`

Reply:
Yeah, I can change that too, while at it.

Reply:
Maybe just a little bit :)

Reply:
Actually, since we already dropped testing of py3.5, should I make this `>=3.6`?